metasrv/src/configs/outer_v0.rs

```diff
@@ -334,6 +337,11 @@ pub struct RaftConfig {
     /// For test only: specifies the tree name prefix
     #[clap(long, default_value = "")]
     pub sled_tree_prefix: String,
+
+    /// Tne node name. If the user specifies a name, the user-supplied name is used,
+    /// if not, the default name is used
+    #[clap(long, default_value = "foo_cluster")]
+    pub name: String,
 }
```

Comment:
All of the default value for a field in inner config, outer config and env config has to be the same. Because the `serfig` crate use the default value to determine if a field is set by user or not.

Reply:
Because the default values in the code are hard-coded, when the user modifies the cluster_name in the configuration file or parameter, serfig will see inconsistencies with the default. The default name in the code is now foo_cluster